blog-service/2023-07-11-saved-slo.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Saved SLOs (Reliability Management)
```

Comment:
We can call it "Saved views for SLOs"

Reply:
Since you're saving the filters, what about something like "Save Custom SLO Filter View"?

Reply:
Hmm. How about "Saved Filter Views for SLOs"? :)

Reply:
yay!!